File: Java/COVID_19.java
import java.util.*;
class COVID_19
{
    public static void main(String args[])
    {
        Scanner in = new Scanner(System.in);
        
        int t=in.nextInt();
        int op[]=new int[t];
        for(int i=0;i<t;i++){
            float n=in.nextFloat();
            float m=in.nextFloat();
            op[i]=(int)((Math.ceil(m/2.0))*(Math.ceil(n/2.0)));
    }
    for(int i=0;i<t;i++)
        System.out.println(op[i]);
    }

}

File: C++ n C/C++/COVID_19.cpp
#include <iostream>
#include <math.h>
using namespace std;
int main(){
    int t;
    cin>>t;
    int op[t];
    for(int i=0;i<t;i++){
        float n,m;
        cin>>n,m;
        //op[i]=(int)(ceil(m/2.0))*(ceil(n/2.0));
        cout>>ceil(m/2.0)>>" ">>ceil(n/2.0);
    }
    for(int i=0;i<t;i++)
        cout<<op[i]<<endl;
}